packages/bot-web-ui/src/components/flyout/help-contents/flyout-video.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const FlyoutVideo = (props: { url: string }) => (
+    <div className='flyout__item' data-testid='dt_flyout_video_container'>
+        <iframe
+            data-testid='dt_flyout_video'
+            className='flyout__video'
+            src={props.url}
+            frameBorder='0'
+            allow='accelerometer; encrypted-media; gyroscope; picture-in-picture'
+            allowFullScreen
+            width={'100%'}
```

Comment:
@maryia-matskevich-deriv any specific reason why we are passing 100% wrapped with curly braces? 
 can we do it like this? width='100%', I know you just converted the file from jsx to tsx but if possible let's do it as well 🙏

Reply:
@sandeep-deriv done✅